sign_oca/security/security.xml

```diff
@@ -38,26 +38,6 @@
         <field name="perm_unlink" eval="0" />
     </record>
     <!-- User Record Rules -->
-    <record id="sign_oca_template_rule_user_read" model="ir.rule">
```

Comment:
Why are you removing the rules?

Reply:
For many reasons: 
- The current rules are incorrect (the user_id field does not exist, it should be create_uid). 
- Really thinking about it I think it is not necessary to filter templates.

Reply:
I see, it seems legit to me!

Thanks!

:+1: